go-filecoin-tutorial/How-to-connect-two-nodes-directly.md

```diff
@@ -1,39 +1,39 @@
 # Connecting two nodes directly
 
-This guide is primarily for getting two nodes mining and connected to a swarm on a single machine, for local development. Some of the instructions can still apply to connecting to a devnet.
+This guide is primarily for getting two nodes mining and connected to a swarm on a single machine for local development purposes. Some of the instructions can still apply to connecting to the testnet.
 
-Optionally, you could look at and modify your own copy of [this script](https://github.com/filecoin-project/go-filecoin/blob/master/functional-tests/retrieval), which sets up two local nodes and does a file store/retrieval.
+It is optionally possible to modify a copy of [this script](https://github.com/filecoin-project/go-filecoin/blob/master/functional-tests/retrieval), which can be used to set up two local nodes and perform a file store/retrieval.
 
 ## Table of contents
 
 - [Initial setup](#initial-setup)
-    - [Node 1](#node-1)
-    - [Node 2](#node-2)
+    - [Node one](#node-one)
+    - [Node two](#node-two)
 
 ## Initial setup
 
-First, set the location of the filecoin source (substitute the path with where ever your source tree is cloned):
+Set the location of the filecoin source (substitute the path the location of the cloned source tree):
```

Comment:
"substitute the path the location" is there a word missing here?

Reply:
Oops! Good catch :slightly_smiling_face: